.github/workflows/tests.yml

```diff
@@ -37,17 +37,33 @@ jobs:
       - name: Install Bundler RubyGems
         run: brew install-bundler-gems --groups=sorbet
 
-      - name: Install shellcheck
-        run: brew install shellcheck
+      - name: Install linting formulae
+        run: brew install shellcheck shfmt vale
 
       - run: brew style --display-cop-names
 
       - run: brew typecheck
 
       - name: Run vale for docs linting
+        run: vale docs/
+
+      - name: Install Ruby for linting tools
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+          working-directory: Library/Homebrew
+
+      - name: Run yard for rubydoc output
+        working-directory: Library/Homebrew
         run: |
-          brew install vale
-          vale docs/
+          bundle install --jobs 4 --retry 3
+          bundle exec yard doc --plugin sorbet --no-output #--fail-on-warning
+
+      - name: Run mdl for docs formatting
+        working-directory: docs
```

Comment:
I think, partly from CI failures, it may be worth separating the `yard` stuff running in `Library/Homebrew` from that running in `docs`. Also, while we're here: perhaps run a `jekyll build` in `docs` too to make sure it doesn't fail?